tests/smoke/block_storage_smoke_test.go

```diff
@@ -15,12 +34,19 @@ func TestBlockStorageSmokeSuite(t *testing.T) {
 type BlockStorageTestSuite struct {
 	suite.Suite
 	helper *SmokeTestHelper
+	rook   *RookHelper
 }
 
 func (suite *BlockStorageTestSuite) SetupTest() {
 	var err error
 
-	suite.helper, err = CreateSmokeTestClient(tests.Platform)
+	suite.rook, err = NewRookHelper()
+	err = suite.rook.InstallRook()
+	if err != nil {
+		panic(fmt.Errorf("failed to install rook: %v", err))
+	}
```

Comment:
we should move Install (and install rook) into base_test init() so we don't install rook for every test run, we shoul djust install rook and  uninstall rook per suite.